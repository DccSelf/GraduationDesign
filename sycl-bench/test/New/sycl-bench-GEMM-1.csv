# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.731079,0.652985,0.644608,"0.644608 0.644668 0.644930 0.645425 0.645889 0.646378 0.646392 0.646936 0.647091 0.647962 0.652985 0.745818 0.765473 0.813215 0.816869 0.819210 0.821790 0.892670 0.907487 0.925780",0.103895,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,32.181593,32.799560,25.518681,"25.518681 26.129257 27.473795 27.535851 28.979416 32.315416 32.550832 32.760642 32.776430 32.783159 32.799560 33.062805 33.067589 33.296341 33.314639 34.998645 35.269527 35.952107 36.170981 36.876179",3.327262,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.571819,0.569485,0.568319,"0.568319 0.568402 0.568412 0.568413 0.568415 0.568754 0.568984 0.569401 0.569430 0.569456 0.569485 0.569526 0.569782 0.570762 0.570785 0.573643 0.576424 0.579728 0.579989 0.588273",0.005306,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,29.032148,29.103776,27.550715,"27.550715 27.575293 27.754397 28.126409 28.154471 28.318563 28.457066 28.724189 28.861713 29.091119 29.103776 29.473467 29.561459 29.623220 29.818673 29.897754 30.016359 30.038071 30.224457 30.271790",0.907393,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.826808,0.870620,0.653247,"0.653247 0.654460 0.655322 0.661044 0.709992 0.817878 0.837578 0.846170 0.847158 0.861534 0.870620 0.871635 0.872564 0.884591 0.889017 0.905196 0.909508 0.910089 0.929435 0.949117",0.100313,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,29.460853,29.772315,26.031377,"26.031377 27.067882 27.140052 28.169946 28.508787 28.720146 28.789928 29.246895 29.538174 29.677598 29.772315 30.001643 30.285670 30.364429 30.646317 30.695187 30.772380 30.959690 31.228892 31.599760",1.504848,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,512,0.589495,0.577321,0.572950,"0.572950 0.572993 0.573346 0.573897 0.574054 0.574748 0.575024 0.575148 0.576554 0.576598 0.577321 0.578152 0.579582 0.581375 0.582177 0.588583 0.594668 0.596425 0.671354 0.694945",0.032964,N/A,UNKNOWN,N/A
# Benchmark name,Verification,device-name,kernel-time-mean,kernel-time-median,kernel-time-min,kernel-time-samples,kernel-time-stddev,kernel-time-throughput,local-size,problem-size,run-time-mean,run-time-median,run-time-min,run-time-samples,run-time-stddev,run-time-throughput,sycl-implementation,throughput-metric
Polybench_Gemm,PASS,SYCL host device,N/A,N/A,N/A,N/A,N/A,N/A,256,1024,28.139781,29.433640,25.335762,"25.335762 25.445495 25.576051 25.592063 25.615723 26.225726 26.307055 26.495993 27.132100 29.006031 29.433640 29.652336 29.703621 29.716286 29.741938 29.787147 29.892148 30.327754 30.533001 31.275745",2.098118,N/A,UNKNOWN,N/A
