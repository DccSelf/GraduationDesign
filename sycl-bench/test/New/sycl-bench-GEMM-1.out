Using test profile: default


##################################################
Processing gemm-nd_range
##################################################
0.0 10800.0
__________________________________________________

gemm-nd_range --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=512 --local=256
==> Benchmark run finished in 27.63965126220137 s
27.63965126220137 10800.0
__________________________________________________

gemm-nd_range --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=1024 --local=256
==> Benchmark run finished in 1231.3233115319163 s


##################################################
Processing gemm-single_task
##################################################
0.0 10800.0
__________________________________________________

gemm-single_task --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=512 --local=256
==> Benchmark run finished in 22.618850288912654 s
22.618850288912654 10800.0
__________________________________________________

gemm-single_task --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=1024 --local=256
==> Benchmark run finished in 1142.7742363102734 s


##################################################
Processing gemm
##################################################
0.0 10800.0
__________________________________________________

gemm --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=512 --local=256
==> Benchmark run finished in 31.59576431196183 s
31.59576431196183 10800.0
__________________________________________________

gemm --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=1024 --local=256
==> Benchmark run finished in 1145.1609407318756 s


##################################################
Processing gemm-hierarchical
##################################################
0.0 10800.0
__________________________________________________

gemm-hierarchical --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=512 --local=256
==> Benchmark run finished in 23.219159338623285 s
23.219159338623285 10800.0
__________________________________________________

gemm-hierarchical --num-runs=20 --output=./sycl-bench-GEMM-1.csv --size=1024 --local=256
==> Benchmark run finished in 1085.1363640064374 s
All benchmarks were executed successfully
